File: R/hello_world.R
#
# Functional R program to print messages
#
# Imports

# Variables
msg <- "Hello World (with no quotation marks)!"

hello_world <- function() {
    print(msg, quote=FALSE);
}

main <- function() {
   hello_world()
}

# Main
if (sys.nframe() == 0) {
    main()
} else {}

File: JavaScript/hello_world.js
// 
// Script for printing and text calls in JavaScript
// 
// Imports
import { fileURLToPath } from "url";


// Variables
const msg = "Hello World!";


// Functions
export function hello_world() { 
    console.log(msg);
}


function main() {
    hello_world();
}


// Main
if (process.argv[1] === fileURLToPath(import.meta.url)) {
    main();
}
